src/ImmutableX.ts

```diff
@@ -350,6 +356,87 @@ export class ImmutableX {
       });
   }
 
+  /**
+   * Get a list of metadata refreshes
+   * @param ethSigner - the L1 signer
+   * @param collectionAddress - the Collection contract address
+   * @param pageSize - the page size of the result
+   * @param cursor - the cursor
+   * @returns a promise that resolves with the requested metadata refreshes
+   * @throws IMXError
+   */
+  public listMetadataRefreshes(
+    ethSigner: EthSigner,
+    collectionAddress?: string,
+    pageSize?: number,
+    cursor?: string,
+  ) {
+    return this.workflows
+      .listMetadataRefreshes(ethSigner, collectionAddress, pageSize, cursor)
+      .then(res => res.data)
+      .catch(err => {
+        throw formatError(err);
+      });
+  }
+
+  /**
+   * Get a list of metadata refresh errors
+   * @param ethSigner - the L1 signer
+   * @param refreshId - the metadata refresh ID
+   * @param pageSize - the page size of the result
+   * @param cursor - the cursor
+   * @returns a promise that resolves with the requested metadata refresh errors
+   * @throws IMXError
+   */
+  public getMetadataRefreshErrors(
+    ethSigner: EthSigner,
+    refreshId: string,
+    pageSize?: number,
+    cursor?: string,
+  ) {
+    return this.workflows
+      .getMetadataRefreshErrors(ethSigner, refreshId, pageSize, cursor)
+      .then(res => res.data)
+      .catch(err => {
+        throw formatError(err);
+      });
+  }
+
+  /**
+   * Get a list of metadata refresh results
+   * @param ethSigner - the L1 signer
+   * @param refreshId - the metadata refresh ID
+   * @returns a promise that resolves with the requested metadata refresh results
+   * @throws IMXError
+   */
+  public getMetadataRefreshResults(ethSigner: EthSigner, refreshId: string) {
+    return this.workflows
+      .getMetadataRefreshResults(ethSigner, refreshId)
+      .then(res => res.data)
+      .catch(err => {
+        throw formatError(err);
+      });
+  }
+
+  /**
+   * Request a metadata refresh
+   * @param ethSigner - the L1 signer
+   * @param request the request object containing the parameters to be provided in the API request
+   * @returns a promise that resolves with the requested metadata refresh ID
+   * @throws IMXError
+   */
+  public requestMetadataRefresh(
```

Comment:
Could this please be `createMetadataRefresh`. This flows in line with `createTrade` and `createOrder`.

Reply:
[6c49010](https://github.com/immutable/imx-core-sdk/pull/175/commits/6c49010930cb947de387c2a65d69188f659f1204)